### PR TITLE
Update GMT canvas

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/basis/paper.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ JPG（推荐）、BMP、PPM 和 TIFF 格式。
 在开始 GMT 绘图时，即可指定要生成的图片格式。例如，下面的示例指明同时生成 PDF 和 PNG 格式的图片::
 
     gmt begin figname pdf,png
-    gmt ...
+        gmt ...
```
Comment (seisman) [inline source/basis/paper.rst:17]: 我比较犹豫，这一节的示例，是按照现在的写法，给出一个框架 （即 ``gmt ...``） 还是给出具体的某个命令（比如 ``gmt basemap``）。

后者的优点在于，用户可以直接执行命令理解其输出。

Comment (core-man) [reply source/basis/paper.rst:17]: 我比较喜欢实际例子，我觉得使用 GMT 的用户应该能举一反三，知道可以适用于其他命令。

File: source/basis/paper.rst [hunk 2]
```diff
@@ -8,52 +8,98 @@
...
+绘图原点
+--------
+
+GMT 在绘图时，默认的绘图原点位于画布左下角 (12.7 cm, 12.7 cm) 处。因而绘图时，
+如果需要移动绘图原点，只能向右和向上移动，而不能向左或向下移动。向左或向下移动绘图原点，
+均可能使得部分图件超过画布左边距或下边界，导致图件被部分截断。
+
+TODO: 这里计划增加一张类似下面的图。
+
+.. figure:: https://docs.gmt-china.org/5.4/_images/paper-gmtplot-1.png
```
Comment (seisman) [inline source/basis/paper.rst:105]: @core-man This PR is almost ready for review. The only missing part is a better figure showing the position of the plot origin on the full-size GMT paper.

I'll work on the image if you think the other parts are all OK.

**Comment by seisman** [comment]
map

I added this figure to explain the relative positioning of plot origin on a "full-size" paper.

File: source/basis/paper.rst [hunk 3]
```diff
@@ -1,59 +1,105 @@
...
 画布大小
 --------
 
-GMT 中默认使用宽 11.55 米、高 11.55 米的画布，绘图原点位于画布左下角。
-在最终生成图片文件时会自动将未使用的白色区域裁减掉，因而用户无需关心画布大小。
+GMT 中默认使用宽 11.6 米、高 11.6 米的画布，在最终生成图片文件时会自动将未使用的白色区域裁减掉，
+因而用户无需关心画布大小。
 
 若需要生成特定尺寸的图片，而不裁剪周边的白色区域，可以修改 GMT 参数
 :term:`PS_MEDIA` 来指定纸张尺寸::
```
Comment (core-man) [inline source/basis/paper.rst:73]:
```suggestion
因而用户无需关心画布大小。若需要生成特定尺寸的图片，而不裁剪周边的白色区域，
可以修改 GMT 参数 :term:`PS_MEDIA` 来指定纸张尺寸::
```

File: source/basis/paper.rst [hunk 4]
```diff
@@ -1,59 +1,105 @@
...
+:贡献者: |田冬冬|
+
+----
+
 要画一张图，首先需要准备一个画布。GMT 默认的画布是一个 PDF 文件。
 
 图片格式
 --------
 
 GMT 支持多种矢量图片格式和位图图片格式。
```
Comment (core-man) [inline source/basis/paper.rst:13]:
```suggestion
GMT 支持多种矢量图片格式和位图图片格式：
```

File: source/basis/paper.rst [hunk 5]
```diff
@@ -1,59 +1,105 @@
...
 
-矢量图片格式包括 PDF（推荐使用）、PS 和 EPS 格式；位图图片格式包括 PNG（推荐）、
-JPG（推荐）、BMP、PPM 和 TIFF 格式。
+矢量图片格式包括 PDF [**推荐**]、PS 和 EPS 格式；位图图片格式包括 PNG [**推荐**]、
+JPG [**推荐**]、BMP、PPM 和 TIFF 格式。
```
Comment (core-man) [inline source/basis/paper.rst:16]:
```suggestion
- 矢量图片格式包括 PDF [**推荐**]、PS 和 EPS 格式；
- 位图图片格式包括 PNG [**推荐**]、JPG [**推荐**]、BMP、PPM 和 TIFF 格式。
```

**Review by core-man — APPROVED** [review]

**Review by core-man — APPROVED** [review]

**Comment by seisman** [comment]
@core-man I renamed the file name from `paper.rst` to `canvas.rst` because "canvas" is the correct name. The PR now is a little difficult to review due to the file name change, so you have to read the whole section again.

The script used to generate the figure is also added to this repository, but the script is neither executed by the gmtplot directive, nor shown in the documentation. It's included in the repo to make sure that we can reproduce the image in the future.

**Comment by seisman** [comment]
Ping @core-man for review.

File: source/basis/canvas.rst [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+    gmt begin figname
+        gmt set PS_PAGE_COLOR lightred
+        gmt basemap -R0/10/0/10 -JX10c -Baf
+    gmt end show
+
+绘图原点
+--------
+
+GMT 在绘图时，默认的绘图原点位于画布（11.6 米宽、11.6 米高）的左下角 (12.7 cm, 12.7 cm) 处
```
Comment (wangliang1989) [inline source/basis/canvas.rst:92]: 初学者可能不理解什么是绘图原点。应该先陈述绘图原点的定义，再说它在哪

Comment (seisman) [reply source/basis/canvas.rst]: Fixed in a18acff4.

File: source/basis/canvas.rst [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+    gmt begin figname
+        gmt set PS_PAGE_COLOR lightred
+        gmt basemap -R0/10/0/10 -JX10c -Baf
+    gmt end show
+
+绘图原点
+--------
+
+GMT 在绘图时，默认的绘图原点（即底图边框左下角）位于画布（11.6 米宽、11.6 米高）的
```
Comment (wangliang1989) [inline source/basis/canvas.rst:92]: **前面解释底图了吗？**
```suggestion
绘图原点是指底图的左下角。默认的绘图原点（即底图边框左下角）位于画布（11.6 米宽、11.6 米高）的
```

**Review by wangliang1989 — CHANGES_REQUESTED** [review]

File: source/basis/canvas.rst (hunk 7 above)
Comment (seisman) [reply source/basis/canvas.rst:92]: 默认读者知道底图

Comment (wangliang1989) [reply source/basis/canvas.rst:92]: 我觉得他很可能不知道何为底图

**Pull request merged**